app/ts/PreferencesTab/Preferences/TradeOff/TradeOff.tsx

```diff
@@ -0,0 +1,47 @@
+import {Grid, Typography} from '@material-ui/core';
+import React, {useContext} from 'react';
+import {PreferencesContext} from '../../PreferencesContext';
+import {TradeOffContextProviderComponent} from './TradeOffContext/TradeOffContext';
+import TradeOffReferenceCriterion from './TradeOffReferenceCriterion/TradeOffReferenceCriterion';
+import TradeOffReferenceCriterionStatement from './TradeOffReferenceCriterionStatement/TradeOffReferenceCriterionStatement';
+import TradeOffTable from './TradeOffTable/TradeOffTable';
+import _ from 'lodash';
+import IPvf from '@shared/interface/Problem/IPvf';
+
+export default function TradeOff(): JSX.Element {
+  const {pvfs} = useContext(PreferencesContext);
+
+  const areAllPvfsLinear = _.every(pvfs, (pvf: IPvf): boolean => {
```

Comment:
closed version has hasNonLinearPvf function in PreferencesUtil. I think I like this version better but maybe would be nice to unify?

Reply:
I also want to check here whether all pvfs are set.